browser/ui/views/side_panel/brave_side_panel_coordinator.h

```diff
@@ -35,9 +35,6 @@ class BraveSidePanelCoordinator : public SidePanelCoordinator {
       const UniqueKey& unique_key,
       SidePanelEntry* entry,
       std::optional<std::unique_ptr<views::View>> content_view) override;
-  void OnEntryWillDeregister(SidePanelRegistry* registry,
```

Comment:
Don't forget to delete the overrides too.

Reply:
ok, will do. 👍🏼